clients/python/client/setup.py

```diff
@@ -1,15 +1,12 @@
-import json
 from pathlib import Path
-from typing import Any, Dict
 
 from setuptools import find_packages, setup  # noqa: H301
 
-repo_root: Path = (Path(__file__) / "../../../..").resolve()
-
-config: Dict[str, Any] = json.loads((repo_root / "api/config.json").read_text())
+VERSION_FILE: Path = Path(__file__).parent / "VERSION"
```

Comment:
Q: so this file does not exists in the repo but is generated in the CI, right? or we can create it using `make client/VERSION`

Reply:
Q: did you add it in the `.gitignore`?

Reply:
Yes, I added a make recipe for this https://github.com/ITISFoundation/osparc-simcore-clients/blob/39b49dd9a9f33365959acb127b5338cefb5efd9d/clients/python/Makefile#L18. I also added it to `.gitignore`